plugin/adsk/scripts/mayaUsdTranslatorExport.mel

```diff
@@ -455,8 +457,10 @@ global proc mayaUsdTranslatorExport_SetFromOptions(string $currentOptions, int $
             }
         }
 
-        // We now know if we export nurbs curves or not:
-        checkBoxGrp -e -v1 $exportNurbsCurves -en $enable exportCurvesCheckBox;
+        // We now know if we export nurbs curves or not and if forced by the job context:
+        if ($contextForcesFilterTypes == 1) {
+            checkBoxGrp -e -v1 $exportNurbsCurves -en $enable exportCurvesCheckBox;

```

Comment:
My understanding is the the NURBS curve Ui shuld *only* be disabled if the job context actually forced the filter types. Otherwise the previously-existing code would always disable that check-box as soon as the user selected a job context in the export dialog.